README.md

```diff
@@ -1,64 +1,44 @@
-| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------| -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [![Build Status](https://travis-ci.org/infobloxopen/terraform-provider-infoblox.svg?branch=master)](https://travis-ci.org/infobloxopen/terraform-provider-infoblox)  | [![Build Status](https://travis-ci.org/infobloxopen/terraform-provider-infoblox.svg?branch=develop)](https://travis-ci.org/infobloxopen/terraform-provider-infoblox) |
-
-## Requirements
-* [Terraform](https://www.terraform.io/downloads.html) 0.11.x or greater
-* [Go](https://golang.org/doc/install) 1.12.x (to build the provider plugin)
-* CNA License need to be installed on NIOS. If CNA is not installed then following default EA's should be added in NIOS side:
-   * VM Name :: String Type
-   * VM ID :: String Type
+## NIOS Requirements
+CNA License need to be installed on NIOS. If CNA is not installed then following default EA's should be added in NIOS side manually:
    * Tenant ID :: String Type
    * CMP Type :: String Type
    * Cloud API Owned :: List Type (Values True, False)
    * Network Name :: String Type
+   * VM Name :: String Type
+   * VM ID :: String Type
 
-## Building the Provider
-
-```sh
-$ git clone https://github.com/infobloxopen/terraform-provider-infoblox
-$ cd terraform-provider-infoblox
-$ make build
-```
-
-## Using the Provider
-If you're building the provider, follow the instructions to [install it as a plugin](https://www.terraform.io/docs/plugins/basics.html#installing-a-plugin). After the build is complete, copy the `terraform-provider-infoblox` binary into the same path as your terraform binary. After placing it into your plugins directory, run `terraform init` to initialize it.
+## Quick Starts
+- [Using the provider](docs/USING.md)
+- [Developing the provider](docs/DEVELOPMENT.md)
 
-## Developing the Provider
-If you wish to work on the provider, you'll first need Go installed on your machine (version 1.12+ is required).
+## Documentation
+The comprehensive documentation of plugin is available at Terraform registry.
 
-To compile the provider, run the following steps:
-```sh
-$ make build
-...
-$ ./terraform-provider-infoblox
-...
-```
-To test the provider, you can simply run `make test`.
-```sh
-$ make test
-```
+https://registry.terraform.io/providers/infobloxopen/infoblox/latest/docs
 
-In order to run the full suite of acceptance tests `make testacc`.
-```sh
-$ make testacc
-```
-## Features of Provider
+## Provider features
+Provider has NIOS DDI resources as Terraform resources and datasources. Below is the consolidated list of the same.
```

Comment:
"Below is the consolidated list of the same." -> "Below is the consolidated list of supported resources:"